src/lib/httpsClient.ts

```diff
@@ -18,6 +18,10 @@ class HttpsClient {
     });
 
     req.on("error", (e) => {
+      console.log('exception: ', e);
+
+      // const RESOLVE_URL = `https://developer.paypay.ne.jp/develop/resolve?api_name=${}&code=${}&code_id=${}`
```

Comment:
when there is an error it should print the console.log with the link.

code and code_id you will you get from the server response the API name you have to fetch from the conf method mapping